bot/resources/tags/venv.md

```diff
@@ -1,21 +1,20 @@
 **Virtual Environments**
 
-Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible.
+Virtual environments are isolated Python environments, which make it easier to keep your system clean and manage dependencies. By default, when activated, only libraries and scripts installed in the virtual environment are accessible, allowing dependency conflicts to be resolved, and your program requirements to be found easily.
 
 To create a new virtual environment, you can use the standard library `venv` module: `python3 -m venv .venv` (replace `python3` with `python` or `py` on Windows)
 
 Then, to activate the new virtual environment:
 
-PowerShell: `.venv\Scripts\Activate.ps1`
-Command Prompt: `.venv\Scripts\activate.bat`
-Bash: `source .venv/bin/activate`
+**Windows** (PowerShell): `.venv\Scripts\Activate.ps1`
+or (Command Prompt): `.venv\Scripts\activate.bat`
+**MacOS / Linux** (Bash): `source .venv/bin/activate`
+
+Note: On Windows, you may need to change the [execution policy](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_execution_policies) first. This is only required once:
+`Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
```

Comment:
I really do think this should be at the very bottom. As it is it somewhat interrupts the flow of the text like an overly large () in the middle of a paragraph.